build.sh

```diff
@@ -95,6 +95,21 @@ if [ "${1}x" == "x" ] || [ "${1}" == "--help" ] || [ "${1}" == "-h" ]; then
   fi
 fi
 
+###
+# Only echos if GH_ACTION is defined
+###
+gh_echo() {
+  if [ -n "${GH_ACTION}" ]; then
+    echo "${@}"
+  fi
+}
+
+gh_env() {
+  if [ -n "${GH_ACTION}" ]; then
+    echo "${@}" >>"$GITHUB_ENV"
+  fi
+}
+
```

Comment:
Maybe these function definitions could be moved to the `build-functions` directory, then we have a consistent structure with the function imports.

Reply:
Great idea!